File: Java/HomeWork06/src/Main.java
void main() {

    int taskNum;
    Scanner input = new Scanner(System.in);
    do {
        System.out.print("Choose a task number [4 - 11] or 0 to exit: ");
        taskNum = input.nextInt();
        if (taskNum == 0) {
            break;
        }

        switch (taskNum) {
            case 4: Task4.run(); break;
            case 5: Task5.run(); break;
            case 6: Task6.run(); break;
            case 7: Task7.run(); break;
            case 8: Task8.run(); break;
            case 9: Task9.run(); break;
            case 10: Task10.run(); break;
            case 11: Task11.run(); break;

            default: System.out.println("Invalid task number");
        }

    } while (true);
}

File: Python/HomeWork06/main.py
import task02
import task03

if __name__ == '__main__':
    tasks = {
        "2": task02.run,
        "3": task03.run,
    }
    tasks_nums = list(tasks.keys())
    while True:
        task_num = input(f"Select task to execute {tasks_nums} or 0 to finish: ")

        if task_num == "0" or not task_num:
            break

        if task_num in tasks:
            tasks[task_num]()
            print()
        else:
            print(f"Invalid task. Select from {tasks_nums}.")